lib/diversity/registry/diversity_api.rb

```diff
@@ -59,8 +56,7 @@ def get_component(name, version = nil)
         begin
           versions = get_installed_versions(name)
         rescue
-          # If the component isn't available by diversity api, let someone else try.
-          return super
+          fail "Failed to get versions"
```

Comment:
I would like to be a bit more specific, ie "Failed to get versions for #{name}."